tidb-cloud/backup-and-restore-serverless.md

```diff
@@ -114,7 +114,8 @@ To restore your TiDB Cloud Starter cluster, follow these steps:
     1. Click **Restore to a New Cluster**.
     2. Enter a name for the new cluster.
     3. Choose the cluster plan for the new cluster.
-    4. If you choose a scalable cluster, set a monthly spending limit, and then configure advanced settings as needed. Otherwise, skip this step.
+       1. If you choose a Starter cluster, set a monthly spending limit, and then configure advanced settings as needed.
```

Comment:
for starter, I think there is no advanced settings. Encryption and HA are not avaiable for starter